tesseract-server/src/handlers/logic_layer/geoservice.rs

```diff
@@ -31,23 +32,16 @@ pub fn query_geoservice(
     geoservice_query: &GeoserviceQuery,
     geo_id: &str
 ) -> Result<Vec<GeoServiceResponseJson>, Error> {
-    let mut query_url: String;
+    let base_url = Url::parse(geoservice_url.clone()).unwrap();
```

Comment:
Let's move the parsing to tesseract startup.
There, it should be easy to return a good error.
Also, parse takes `&str`, so I don't think you have to clone?